### PR TITLE
Remove Django dependency from vumitools

### PR DESCRIPTION
**Pull request by jerith**
We use `django.utils.datastructures.SortedDict` in `go.vumitools.api` which pulls Django into places it shouldn't be.

### CODE REVIEW

File: go/conversation/forms.py [hunk 1]
```diff
@@ -10,8 +10,9 @@ class ConversationDetailForm(forms.Form):
 class NewConversationForm(ConversationDetailForm):
     def __init__(self, user_api, *args, **kwargs):
         super(NewConversationForm, self).__init__(*args, **kwargs)
+        apps = [v for k, v in sorted(user_api.applications().iteritems())]
```
Comment (rudigiesler) [inline go/conversation/forms.py]: Can we not make this a generator instead of a list, since we're turning it into a list below?

**Comment by rudigiesler** [comment]
Just the one comment (which would apply to all the instances where that pattern was used).

File: go/conversation/forms.py (hunk 1 above)
Comment (jerith) [inline go/conversation/forms.py]: We need the whole source list in-memory for sorting anyway, but making it a generator won't hurt.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**